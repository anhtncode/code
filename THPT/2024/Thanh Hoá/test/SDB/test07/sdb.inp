10
10 280472
589864 606510
497063 632421
2 394226
7 904321
3 151342
5 180299
279868 714412
192824 386309
420128 793775
