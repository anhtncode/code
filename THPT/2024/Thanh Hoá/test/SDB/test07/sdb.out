40
1
5
46
65
31
34
20
12
15
